matrix/Vector.hpp

```diff
@@ -83,6 +83,14 @@ class Vector : public Matrix<Type, M, 1>
         return (*this) / norm();
     }
 
+    Vector unit_or_zero(const Type eps = 1e-5f) {
+        const Type n = norm();
+        if (n > eps) {
+            return (*this) / norm();
```

Comment:
`return (*this) / n;`

Reply:
This was my intention, don't know how it got lost and unnoticed... fixed now.